scripts/west_commands/sign.py

```diff
@@ -423,11 +423,12 @@ class RimageSigner(Signer):
     def sign(self, command, build_dir, build_conf, formats):
         args = command.args
 
-        if args.tool_path:
-            command.check_force(shutil.which(args.tool_path),
-                                '--tool-path {}: not an executable'.
-                                format(args.tool_path))
-            tool_path = args.tool_path
+        tool_path = args.tool_path if args.tool_path else config_get('rimage', 'path', None)
```

Comment:
... i.e. replace this with `command.config.get('rimage.path')` instead of `config_get`